### PR TITLE
Unity: add connect timeout exception

### PR DESCRIPTION
**Pull request by Murray-LIANG**
Set the default connect timeout to 10 sec for `UnityRESTConnector`. And
raise `StoropsConnectTimeoutError` when timeout to connect to the server
using `HTTPClient`.

### CODE REVIEW

File: storops/connection/connector.py [hunk 1]
```diff
@@ -54,7 +54,8 @@ class UnityRESTConnector(object):
     }
 
     def __init__(self, host, port=443, user='admin', password='',
-                 verify=False, retries=None, cache_interval=0):
+                 verify=False, retries=None, cache_interval=0,
+                 connect_timeout=10):
```
Comment (amagicsun) [inline storops/connection/connector.py]: Why this change?

Comment (Murray-LIANG) [reply storops/connection/connector.py]: It is for the replication feature. When DR happens, Unity could be totally unable to connect. The `connect_timeout` gives storops a quick fail and storops will raise a specific exception. Then OpenStack driver knows the Unity is down.

Comment (amagicsun) [reply storops/connection/connector.py]: I am not sure whether 10 seconds are enough in some extreme cases. Does it impact all the requests to Unity?

Comment (Murray-LIANG) [reply storops/connection/connector.py]: Actually, there are several timeout. `connect_timeout` is only one of them and it is only for the time of setting up the connection, doesn't include data reading and writing time.

Yes, all requests to Unity are impacted.

Comment (amagicsun) [reply storops/connection/connector.py]: If the connection cannot be connected in 10 seconds, storops will timeout?
Would it be too short? 
Please think about 30 seconds or even 60 seconds. I believe normally it is not hit.

Comment (Murray-LIANG) [reply storops/connection/connector.py]: OK. Set it to 30 sec.
